.github/workflows/build.yml

```
@@ -97,18 +81,36 @@ jobs:
           echo "REVISION_VERSION=${REVISION_VERSION}" >> $GITHUB_ENV
           echo "BUILD_VERSION=${BUILD_VERSION}" >> $GITHUB_ENV
 
-      - name: Build and push Docker image
+      - name: Build Docker Image
         run: |
           sbt "set version := \"${{ env.BUILD_VERSION }}\"" "docker:stage"
-          docker buildx build --platform=linux/arm64,linux/amd64 --push -t ghcr.io/hyperledger/identus-cloud-agent:${{ env.BUILD_VERSION}} ./cloud-agent/service/server/target/docker/stage
+          docker buildx build --platform=linux/arm64,linux/amd64 --push -t ${{ vars.DOCKERHUB_ORG }}/identus-cloud-agent:${{ env.BUILD_VERSION }} ./cloud-agent/service/server/target/docker/stage
```

you missing the docker.io
```suggestion
          docker buildx build --platform=linux/arm64,linux/amd64 --push -t docker.io/${{ vars.DOCKERHUB_ORG }}/identus-cloud-agent:${{ env.BUILD_VERSION }} ./cloud-agent/service/server/target/docker/stage
```

---

@FabioPinheiro, afaik, docker.io is a default registry for the docker command
I tested this in the build.yml workflow [here](https://github.com/hyperledger-identus/cloud-agent/blob/ci/1514/.github/workflows/build.yml#L87)